src/index.js

```diff
@@ -65,9 +65,15 @@ const defaults = {
   swagger2ConverterUrl: "https://converter.swagger.io/api/convert",
 }
 
-export default function SwaggerEditor(options) {
+export default function SwaggerEditor(options, swaggerEditorConfigUrls = null) {
```

Comment:
What would be the reason for extending the `SwaggerEditor` function with another parameter called `swaggerEditorConfigUrls`?

`SwaggerEditor` function already supports following configuration options via `options` parameter ([see code here](https://github.com/swagger-api/swagger-editor/blob/2581d92dcf984bea0b6c19d96894f5f24dd1f03a/src/index.js#L63)):

```js
  swagger2GeneratorUrl: "https://generator.swagger.io/api/swagger.json",
  oas3GeneratorUrl: "https://generator3.swagger.io/openapi.json",
  swagger2ConverterUrl: "https://converter.swagger.io/api/convert",
```

So IMHO in order to support new ENV variables we don't need to make any changes to the code.

Reply:
Also updated the PR description.